### ISSUE TITLE
Add release builds

### ISSUE THREAD

**Issue by vladkalyuzhny** [lobobrowser/LoboBrowser#3]
Hi! I have successfully built a jar file with browser. Can you add some artifacts in release tab on github? Because i simply wanted to see in action this project and it wiil be good to have a such opportunity in future. [Download page](https://lobobrowser.org/download) of site is not responding also. I have attached here my artifact for those who else wants to see this app (it is needed to rename file extension from `zip` to `jar` because github hosting doesn't allow to store executable extensions)
[LoboBrowser.zip](https://github.com/lobobrowser/LoboBrowser/files/3052867/LoboBrowser.zip)

**Comment by zhivko** [comment]
Great - thank you I just tried to build project and it doesn't build - instead I get:

```
~/workspace/LoboBrowser$ mvn install
[INFO] Scanning for projects...
[INFO]                                                                         
[INFO] ------------------------------------------------------------------------
[INFO] Building LoboBrowser 1.0-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[WARNING] The POM for org.cobraparser:Cobra:jar:1.0-SNAPSHOT is missing, no dependency information available
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 0.470 s
[INFO] Finished at: 2019-05-28T11:33:18+02:00
[INFO] Final Memory: 10M/240M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project LoboBrowser: Could not resolve dependencies for project org.lobobrowser:LoboBrowser:jar:1.0-SNAPSHOT: Failure to find org.cobraparser:Cobra:jar:1.0-SNAPSHOT in http://oss.sonatype.org/content/groups/public/ was cached in the local repository, resolution will not be reattempted until the update interval of Sonatype-public has elapsed or updates are forced -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
klemen@mint-kz:~/workspace/LoboBrowser$ mvn clean install -U
[INFO] Scanning for projects...
[INFO]                                                                         
[INFO] ------------------------------------------------------------------------
[INFO] Building LoboBrowser 1.0-SNAPSHOT
[INFO] ------------------------------------------------------------------------
Downloading: http://oss.sonatype.org/content/groups/public/org/cobraparser/Cobra/1.0-SNAPSHOT/maven-metadata.xml
Downloading: http://oss.sonatype.org/content/groups/public/org/cobraparser/Cobra/1.0-SNAPSHOT/Cobra-1.0-SNAPSHOT.pom
[WARNING] The POM for org.cobraparser:Cobra:jar:1.0-SNAPSHOT is missing, no dependency information available
Downloading: http://oss.sonatype.org/content/groups/public/org/yaml/snakeyaml/1.19-SNAPSHOT/maven-metadata.xml
Downloaded: http://oss.sonatype.org/content/groups/public/org/yaml/snakeyaml/1.19-SNAPSHOT/maven-metadata.xml (2 KB at 1.0 KB/sec)
Downloading: http://oss.sonatype.org/content/groups/public/org/cobraparser/Cobra/1.0-SNAPSHOT/Cobra-1.0-SNAPSHOT.jar
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 4.407 s
[INFO] Finished at: 2019-05-28T11:33:35+02:00
[INFO] Final Memory: 12M/291M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project LoboBrowser: Could not resolve dependencies for project org.lobobrowser:LoboBrowser:jar:1.0-SNAPSHOT: Could not find artifact org.cobraparser:Cobra:jar:1.0-SNAPSHOT in Sonatype-public (http://oss.sonatype.org/content/groups/public/) -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException

```

**Comment by xaoseric** [comment]
> Great - thank you I just tried to build project and it doesn't build - instead I get:
> 
> ```
> ~/workspace/LoboBrowser$ mvn install
> [INFO] Scanning for projects...
> [INFO]                                                                         
> [INFO] ------------------------------------------------------------------------
> [INFO] Building LoboBrowser 1.0-SNAPSHOT
> [INFO] ------------------------------------------------------------------------
> [WARNING] The POM for org.cobraparser:Cobra:jar:1.0-SNAPSHOT is missing, no dependency information available
> [INFO] ------------------------------------------------------------------------
> [INFO] BUILD FAILURE
> [INFO] ------------------------------------------------------------------------
> [INFO] Total time: 0.470 s
> [INFO] Finished at: 2019-05-28T11:33:18+02:00
> [INFO] Final Memory: 10M/240M
> [INFO] ------------------------------------------------------------------------
> [ERROR] Failed to execute goal on project LoboBrowser: Could not resolve dependencies for project org.lobobrowser:LoboBrowser:jar:1.0-SNAPSHOT: Failure to find org.cobraparser:Cobra:jar:1.0-SNAPSHOT in http://oss.sonatype.org/content/groups/public/ was cached in the local repository, resolution will not be reattempted until the update interval of Sonatype-public has elapsed or updates are forced -> [Help 1]
> [ERROR] 
> [ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
> [ERROR] Re-run Maven using the -X switch to enable full debug logging.
> [ERROR] 
> [ERROR] For more information about the errors and possible solutions, please read the following articles:
> [ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
> klemen@mint-kz:~/workspace/LoboBrowser$ mvn clean install -U
> [INFO] Scanning for projects...
> [INFO]                                                                         
> [INFO] ------------------------------------------------------------------------
> [INFO] Building LoboBrowser 1.0-SNAPSHOT
> [INFO] ------------------------------------------------------------------------
> Downloading: http://oss.sonatype.org/content/groups/public/org/cobraparser/Cobra/1.0-SNAPSHOT/maven-metadata.xml
> Downloading: http://oss.sonatype.org/content/groups/public/org/cobraparser/Cobra/1.0-SNAPSHOT/Cobra-1.0-SNAPSHOT.pom
> [WARNING] The POM for org.cobraparser:Cobra:jar:1.0-SNAPSHOT is missing, no dependency information available
> Downloading: http://oss.sonatype.org/content/groups/public/org/yaml/snakeyaml/1.19-SNAPSHOT/maven-metadata.xml
> Downloaded: http://oss.sonatype.org/content/groups/public/org/yaml/snakeyaml/1.19-SNAPSHOT/maven-metadata.xml (2 KB at 1.0 KB/sec)
> Downloading: http://oss.sonatype.org/content/groups/public/org/cobraparser/Cobra/1.0-SNAPSHOT/Cobra-1.0-SNAPSHOT.jar
> [INFO] ------------------------------------------------------------------------
> [INFO] BUILD FAILURE
> [INFO] ------------------------------------------------------------------------
> [INFO] Total time: 4.407 s
> [INFO] Finished at: 2019-05-28T11:33:35+02:00
> [INFO] Final Memory: 12M/291M
> [INFO] ------------------------------------------------------------------------
> [ERROR] Failed to execute goal on project LoboBrowser: Could not resolve dependencies for project org.lobobrowser:LoboBrowser:jar:1.0-SNAPSHOT: Could not find artifact org.cobraparser:Cobra:jar:1.0-SNAPSHOT in Sonatype-public (http://oss.sonatype.org/content/groups/public/) -> [Help 1]
> [ERROR] 
> [ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
> [ERROR] Re-run Maven using the -X switch to enable full debug logging.
> [ERROR] 
> [ERROR] For more information about the errors and possible solutions, please read the following articles:
> [ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
> ```
It is failing because you need to build the Cobra Parser as well, you can get that from https://github.com/lobobrowser/Cobra

**Comment by xaoseric** [comment]
> Hi! I have successfully built a jar file with browser. Can you add some artifacts in release tab on github? Because i simply wanted to see in action this project and it wiil be good to have a such opportunity in future. [Download page](https://lobobrowser.org/download) of site is not responding also. I have attached here my artifact for those who else wants to see this app (it is needed to rename file extension from `zip` to `jar` because github hosting doesn't allow to store executable extensions)
> [LoboBrowser.zip](https://github.com/lobobrowser/LoboBrowser/files/3052867/LoboBrowser.zip)

Yes, I will be uploading the releases to the site soon as well as a Mac OS X release.

**Comment by ghost** [comment]
how to build pls help idk where to put cobra

**Comment by xaoseric** [comment]
> how to build pls help idk where to put cobra

You don't put Cobra anywhere, it gets shaded in at build time.
